### PR TITLE
Support for etags files

### PR DESCRIPTION
**Pull request by coot**
Related to #16

### CODE REVIEW

File: lib/Plugin/GhcTags/ETags/Formatter.hs [hunk 1]
```diff
@@ -0,0 +1,82 @@
...
+      Just lineNo -> flip BuilderWithSize (tagSize lineNo) $
+        -- TODO: get access to the original line or pretty print original
+        -- declaration
+           tagNameBuilder
+        <> BB.charUtf8 '\DEL' -- or '\x7f'
+        <> tagNameBuilder
+        <> BB.charUtf8 '\SOH' -- or '\x01'
+        <> BB.intDec lineNo
+        <> BB.charUtf8 ','
+        <> BB.intDec (succ lineNo)
```
Comment (arybczak) [inline lib/Plugin/GhcTags/ETags/Formatter.hs]: This should be a byte offset from the source file (I know hasktags also does the same thing as here, but that's wrong).

Comment (coot) [reply lib/Plugin/GhcTags/ETags/Formatter.hs]: Thanks - indeed I wasn't sure if this is right.  I think I can only estimate it with line number and column number.

What emacs does when the offsets are wrong?

Comment (arybczak) [reply lib/Plugin/GhcTags/ETags/Formatter.hs]: If there are multiple tags with the same name in one file (e.g. two or more data types that have a field `name` with `DuplicateRecordFields`), emacs will jump only to the first one.

Comment (arybczak) [reply lib/Plugin/GhcTags/ETags/Formatter.hs]: > I can only estimate it with line number and column number.

Hmm, `hpm_src_files` from HsParsedModule doesn't have path to the file being parsed?

Comment (coot) [reply lib/Plugin/GhcTags/ETags/Formatter.hs]: Yes we can use `ModSummary` to get the file path.

**Comment by coot** [comment]
@arybczak could you test this branch if it works for you?

**Pull request merged**